src/static-build/loader.ts

```diff
@@ -0,0 +1,155 @@
+import getFeatures from './getFeatures';
+import { LoaderContext } from 'webpack/lib/webpack';
+const { getOptions } = require('loader-utils');
+import * as recast from 'recast';
+const types = recast.types;
```

Comment:
stylistically we tend to keep the imports in one block, line break, and then code...